source/docs/articles/sites/create.md

```diff
@@ -1,26 +1,41 @@
 ---
 title: Creating Sites
-description:  How to create a new site on the platform.
+description:  How to create sites on the Pantheon platform.
 category:
   - managing
-keywords: create a site, creating new sites, how to create a site, creating sites, how to create a site on pantheon, create new site on pantheon, pantheon account
+keywords: create a site, how to create a site, creating sites, how to create a site on pantheon, import a site
 ---
-On your User Dashboard, click  **Create your first site** or **Create a new site** to access **[sites/create](https://dashboard.pantheon.io/sites/create)**.
-### Name Your Site
-Choose a name for your site. You cannot change the name of a site after creating it. A development URL will be automatically generated for you, at our pantheon.io domain.
-![Name your site](/source/docs/assets/images/name-your-site.png)
-### Choose Your Framework
-Click **Install WordPress**  or **Install Drupal**  or any of the distributions available on the Platform.
-![Choose your start state](/source/docs/assets/images/core-startup.png)
-### Visit the Dev Installation
-On the Site Dashboard, click **Visit Development Site** to open the Development environment.  You'll then be prompted to complete the Drupal or WordPress installation forms.  
-
-![Visit development site button](/source/docs/assets/images/visit-development-site.png)
-To start adding code or themes, start  with the SFTP connection, or toggle to Git mode to start cloning and pushing.
-## Free Sites
-Free sites are useful for trying out the Pantheon platform, creating sandboxes for development, or for starting a new client project. We allocate two free sandbox sites for all user accounts. When one of your sites converts to a paid plan, you can start another sandbox site. If you need additional development sites, contact us and we'll be happy to work with you.
-## Your Pantheon Account
-Your account is your own individual account, and every account can manage multiple projects (or sites) at a time. Pantheon doesn't recommend sharing your account with other people. If you're collaborating on a project or handing over ownership to a client, use our  team management and ownership transfer tools.  
-## Next Steps
+There are three ways that you can create a site:  
```

Comment:
Why remove the link to https://dashboard.pantheon.io/sites/create? Rule of thumb: when we can link to the dashboard, we link to the dashboard.